### PR TITLE
{vis} [intel/2016b] Bump Mesa to 12.0.2 (REVIEW)

### PR DESCRIPTION
**Pull request by wpoely86**

### CODE REVIEW

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 0 out of 2 (2 easyconfigs in this PR)
node2025.delcatty.os - Linux SL 6.8, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/2d0fd3a6c0c216f8c6e67584e6f2814a for a full test report.

**Comment by boegel** [comment]
not a big fan of bumping versions for stuff that's already merged, but since this is very recent (Mesa 12.0.1 was merged just 4 days ago), I'll let it pass (if it works)

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2694.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/a50c0619fb94f5db070c1979b8eb8caf for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2694.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/aa22899da6bc5ff71e764a7583a6b8f7 for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2025.delcatty.os - Linux SL 6.8, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/af38202313c1d7ec4a1f52d51059d9af for a full test report.

File: easybuild/easyconfigs/m/Mesa/Mesa-12.0.2-intel-2016b.eb [hunk 1]
```diff
@@ -8,13 +8,14 @@
...
 name = 'Mesa'
-version = '12.0.1'
+version = '12.0.2'
 
 homepage = 'http://www.mesa3d.org/'
 description = """Mesa is an open-source implementation of the OpenGL specification -
  a system for rendering interactive 3D graphics."""
 
 toolchain = {'name': 'intel', 'version': '2016b'}
+toolchainopts = {'optarch': False}
```
Comment (boegel) [inline easybuild/easyconfigs/m/Mesa/Mesa-12.0.2-intel-2016b.eb]: @wpoely86 please add a comment to clarify why you're doing this

other than that: good to go!

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Going in, thanks @wpoely86!

**Pull request merged**